### PR TITLE
Add many more modules

### PR DESCRIPTION
**Pull request by mamercad**
Add many more modules; I'd like to have playbook for all of the new modules (but, it's time to land this, too).

### CODE REVIEW

**Review by artis3n — APPROVED** [review]
Is it just me or are the status checks on this PR broken? Like the UI of them - I can't see them from the PR but I see them under Checks, but they show as having been running for 10 minutes but have no details underneath them.

File: .github/workflows/pull-request-integration.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ on:
   pull_request:
 
 concurrency:
-  group: ${{ github.workflow }}
-  cancel-in-progress: false
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
```
Comment (artis3n) [inline .github/workflows/pull-request-integration.yml:8]: Oh thank god

Comment (mamercad) [reply .github/workflows/pull-request-integration.yml:8]: :lol:

**Review by artis3n — DISMISSED** [review]
This PR looks broken to me (from the UI perspective, not the code) but 🚢 
image

**Review by artis3n — APPROVED** [review]

**Pull request merged**